source/js/components/petition/donation-modal.jsx

```diff
@@ -112,6 +128,10 @@ class DonationModal extends Component {
     );
   }
 
+  userElectedCloseModal() {
+    this.props.close();
```

Comment:
typo?
```suggestion
    this.props.onClose();
```

Reply:
Oops! Yes!

Reply:
Oops! Yes! I created this function last, tested it, and everything worked fine. I was the new script was still building when I tested it out. Thanks for spotting it!